apis/nucleus/src/components/listbox/components/grid-list-components/derive-options.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable import/prefer-default-export */
+const FREQUENCY_MIN_SHOW_WIDTH = 80;
+
+export const getFrequencyAllowed = ({ width, layout, frequencyMode }) => {
```

Comment:
```suggestion
const getFrequencyAllowed = ({ width, layout, frequencyMode }) => {
```
then at end of file:
```
export default getFrequencyAllowed;
```
and eslint-disable can be removed

Reply:
I was planning to add more functions here but might not be necessary so I should probably follow your suggestion 👍